keps/sig-storage/20190117-SkipVolumeOwnershipChange.md

```diff
@@ -0,0 +1,147 @@
+---
+title: Skip Volume Ownership Change
+authors:
+  - "@mattsmithdatera"
+  - "@gnuified"
+owning-sig: sig-storage
+participating-sigs:
+reviewers:
+  - "@msau42"
+  - "@liggit"
+approvers:
```

Comment:
I think Michelle or Saad or Jan should be approver.

Reply:
okay, we can add saad.

Reply:
saad has been added